conf/messages

```diff
@@ -663,11 +663,13 @@ hts.reminder-frequency.25=On the 25th day of each month
 hts.reminder-frequency.both=On the 1st day and 25th day of each month
 hts.reminder-frequency.or= or
 hts.reminder-frequency.cancel=Cancel all reminders
-hts.reminder-confirmation.title.h1=Savings reminders set
-hts.reminder-confirmation.title.p1=You’ll now get monthly email reminders
-hts.reminder-confirmation.title.p1-1=You’ll get a Help to Save email reminder on the
+hts.reminder-confirmation_set.title.h1=Savings reminders set
+hts.reminder-confirmation_set.title.p1=You will now get monthly email reminders
+hts.reminder-confirmation_update.title.h1=Savings reminders changed
+hts.reminder-confirmation_update.title.p1=You have new dates for monthly email reminders
```

Comment:
I don't think those should be "_", better "-" like the rest, so: hts.reminder-confirmation-set.title.h1 etc.